src/openrct2/util/util.c

```diff
@@ -396,6 +396,37 @@ char *safe_strtrimleft(char *destination, const char *source, size_t size)
     return safe_strcpy(destination, source, size);
 }
 
+char* stristr(const char* haystack, const char* needle)
```

Comment:
Our code style is now to have spaces around the asterisk - e.g. `type * ptr`. Even if surrounding code doesn't follow this, new code should. There are a few instances that need changing in this function.